### ISSUE TITLE
Duplicate entries in import menu options

### ISSUE THREAD

**Issue by Xrayez** [godotengine/godot#32121]
**Godot version:**
3.2-alpha, 24e1039eb

**Issue description:**
For each type of import, there are duplicate entries:

![duplicate_import_options](https://user-images.githubusercontent.com/17108460/64862677-cb13d580-d63b-11e9-8c21-b37fd63e3538.png)

**Steps to reproduce:**
Just select any resource from the filesystem and go to import tab.

**Comment by AlexHolly** [comment]
I am not able to reproduce on https://github.com/godotengine/godot/commit/680bcb1ef45c67da1b58ff3be84ca5e4fd1e40cf

**Comment by Xrayez** [comment]
Can't reproduce either, because this time I compiled the default version of the engine!

Figured this is caused by [adding indexed image format loader](https://github.com/Xrayez/godot-imagetools/blob/c7fb29cc2c289cfbba33762e82d55a796dacaadd/register_types.cpp#L16) in one of my modules. Both regular and indexed type of imports recognize the same resource as `png`, hence the duplicate options. I'll need to look how to resolve this, the straightforward fix would be not registering the loader at all.

Sorry to bother, closing.
